.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: validate changes
+on:
+  pull_request:
+jobs:
+  validate:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version:
+        - '3.8'
+        TOX_ENV:
+        - csslint
+        - eslint
+        - quality
+        - django22
+        - django30
+        - django31
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
```

Comment:
Is this v2 of the step, or v2 of python?

Reply:
Kinda assuming it's the script, not python.

Reply:
It's the version of the Action to be used. Similar to `pip install -e` with a Github path, you're specifying which version or Git Commit/branch/tag to be checkout out.